### ISSUE TITLE
Disabled ToolBar button does not have aria-disabled attribute

### ISSUE THREAD

**Issue by valchev** [telerik/kendo-ui-core#1596]
### Bug report
Disabled toolbar button does not have aria-disabled attribute. Both item rendering options are affected - as an anchor or a button through useButtonTag: true.

### Reproduction
1. Use API demo: http://demos.telerik.com/kendo-ui/toolbar/api
1. Enable/disable the 'repeat' button

### Current behavior
'Repeat' button does not receive `aria-disabled` attribute when disabled

###Expected behavior
'Repeat' button has `aria-disabled="true"` attribute and `disabled="disabled"` attribute when disabled, and `aria-disabled="false"` attribute when enabled.

### Environment
- **Kendo UI:** 2019.3.1023
- **Browser:** all

**Comment by Dimitar-Goshev** [comment]
Hi @GaloisGirl ,
we are currently reviewing the accessibility issues of ToolBar and considering when there will be available resources for addressing the issues altogether as a whole story. Currently we cannot provide a time estimate for this fix.
